pyspecProcScripts/clock_correct.py

```diff
@@ -38,7 +38,8 @@ def clock_correct(s, fl=None):
         fl.next("clock correction")
         fl.plot(s_clock, ".", alpha=0.7)
     clock_corr = s_clock.argmax("clock_corr").item()
-    plt.axvline(x=clock_corr, alpha=0.5, color="r")
+    if fl is not None:
+        plt.axvline(x=clock_corr, alpha=0.5, color="r")
```

Comment:
I turned off the diagnostic plot for clock correction, which I think we usually don't want to see, and this line kept showing up.